templates/api/qkm-certs.yaml

```
@@ -0,0 +1,16 @@
+{{- if .Values.global.tls.enabled }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: quorumkeymanager-certs
+  labels:
+    {{- include "orchestrate.labels" . | nindent 4 }}
+  annotations:
+    {{- include "orchestrate.annotations" . | nindent 4 }}
+type: Opaque
+data:
+  ca.crt: {{ .Values.global.tls.client.ca }}
```

Dealing with plain text values is easier for the implementation. Could we apply the b64enc here?
```suggestion
  ca.crt: {{ .Values.global.tls.client.ca | b64enc | quote }}
```
Same below.

---

Yes good point